modules/dynamic/test/src-jvm/smithy4s/dynamic/EnumSpec.scala

```diff
@@ -112,14 +112,14 @@ class EnumSpec extends DummyIO.Suite {
         EnumValue(
           stringValue = "Ice",
           intValue = 0,
-          value = 0,
+          value = "Ice",
           name = "ICE",
           hints = Hints.empty
         ),
         EnumValue(
```

Comment:
related to comment on reordering, here, we're _lucky_ that we used name params, otherwise this would have gone under the rader

Reply:
Named params are a life-saver.